### PR TITLE
Donator Item, Smol42

### PR DESCRIPTION
**Pull request by Smol42**
## About The Pull Request

Adds donator item for Smol42 sprited from scratch by Smol42.

<details>
<summary>Proof of Testing</summary>

-=q
</details>

## Changelog

:cl: Smol42
add: Added custom donator item.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1739,3 +1739,26 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
...
 	worn_icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_y_offset = 4
+
+//  Donator reward Smol42
+
+/obj/item/clothing/neck/trenchcoat
+	name = "Graceful Trenchcoat"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/cloaks.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/neck.dmi'
+	icon_state = "trenchcoat"
```
Comment (vinylspiders) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	icon_state = "trenchcoat"
	body_parts_covered = CHEST|GROIN|LEGS|ARMS
```
Same ones as for cloaks

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Otherwise looks good

**Comment by Smol42** [comment]
> Otherwise looks good

👍🏻

**Review by vinylspiders — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.

**Pull request merged**